.gitignore

```diff
@@ -7,6 +7,6 @@
 /cephalopod/static/locale/*.js
 .*.swp
 *.pyc
-*.cfg
+settings.cfg
```

Comment:
When using your own config file you specify the path to the file in an env var so you can call it whatever you want